generated/python/gapic-google-monitoring-v3/google/cloud/gapic/monitoring/v3/enums.py

```diff
@@ -0,0 +1,288 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Wrappers for protocol buffer enum types."""
+
+
+class ValueType(object):
+    """
+    Value types that can be used as label values.
+
+    Attributes:
+      STRING (int): A variable-length string. This is the default.
+      BOOL (int): Boolean; true or false.
+      INT64 (int): A 64-bit signed integer.
+    """
+    STRING = 0
+    BOOL = 1
+    INT64 = 2
+
+
+class MetricKind(object):
+    """
+    The kind of measurement. It describes how the data is reported.
+
+    Attributes:
+      METRIC_KIND_UNSPECIFIED (int): Do not use this default value.
+      GAUGE (int): An instantaneous measurement of a value.
+      DELTA (int): The change in a value during a time interval.
+      CUMULATIVE (int): A value accumulated over a time interval.  Cumulative
+        measurements in a time series should have the same start time
+        and increasing end times, until an event resets the cumulative
+        value to zero and sets a new start time for the following
+        points.
+    """
+    METRIC_KIND_UNSPECIFIED = 0
+    GAUGE = 1
+    DELTA = 2
+    CUMULATIVE = 3
+
+
+class ValueType(object):
```

Comment:
Sorry, I should have asked you to hold off a bit. We need to merge https://github.com/googleapis/toolkit/issues/511 to fix duplicate enum types (see two instances of `ValueType`), so we'll need to re-gen GAPIC 0.10.1 for Python.

I can do this if you give me write access to your fork.